### PR TITLE
Material UI and Login component

### PR DESCRIPTION
**Pull request by rumana-aafrin**
Installed  Material UI

### CODE REVIEW

File: packages/client/.eslintrc [hunk 1]
```diff
@@ -15,6 +15,12 @@
     "react/state-in-constructor": "off",
     "react/jsx-props-no-spreading": "off",
     "react/function-component-definition": "off"
+    // "import/no-extraneous-dependencies" : [
```
Comment (shpomp) [inline packages/client/.eslintrc:18]: Just remove this commented code🙏🏽

File: packages/client/src/App.js [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+// import { LandingPage } from './containers/LandingPage/LandingPage.Container';
+// import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
 
 function App() {
   return (
     <div className="app">
-      <Router>
+      <Login />
+      Hello
+      {/* <Router>
```
Comment (shpomp) [inline packages/client/src/App.js:13]: I would advice to use the routes from the get go, do not comment them out, so you could now just go and use your Login component for test (and maybe later too) in the LandingPage container.

**Review by shpomp — CHANGES_REQUESTED** [review]
Good job and good patience with our troubleshooting! 😅 ⭐ 

I left a few comments for simple cleanup/refactor.
 
Another note is that there is a bit of a challenge with folder structure. You have put your new components in a new "component" folder, but you should rather utilize the existing structure. 
See there is a folder "`src/components`" and, for example, the `Button` is there in the "`Button`" folder. 
So create respective folders in "`src/components`" and move your stuff there 🙏🏽 
"`src/components/Login`" ➡️  for Login.jsx
"`src/components/TeamMembers`" ➡️  for TeamMembers.jsx

**Comment by rumana-aafrin** [comment]
> Good job and good patience with our troubleshooting! 😅 ⭐
> 
> I left a few comments for simple cleanup/refactor.
> 
> Another note is that there is a bit of a challenge with folder structure. You have put your new components in a new "component" folder, but you should rather utilize the existing structure. See there is a folder "`src/components`" and, for example, the `Button` is there in the "`Button`" folder. So create respective folders in "`src/components`" and move your stuff there 🙏🏽 "`src/components/Login`" ➡️ for Login.jsx "`src/components/TeamMembers`" ➡️ for TeamMembers.jsx

**Comment by rumana-aafrin** [comment]
Yessss!  Thanks for saving my day🙏🏽
I was not sure of how to manage it in the components folder. Now I got It!

File: packages/client/.eslintrc (hunk 1 above)
Comment (rumana-aafrin) [reply packages/client/.eslintrc:18]: Done

Comment (ddobby94) [reply packages/client/.eslintrc:18]: I can still see the commented code, maybe you forgot to push it? 🤔

File: packages/client/src/App.js (hunk 2 above)
Comment (ddobby94) [reply packages/client/src/App.js:13]: Exactly, I second @shpomp 😄

**Comment by ddobby94** [comment]
Nice job @rumana-aafrin! Good to see the progress 😄 
And thanks for @shpomp for looking into the PR 🙏

File: packages/client/.eslintrc (hunk 1 above)
Comment (rumana-aafrin) [reply packages/client/.eslintrc:18]: Yes!

**Pull request closed without merging**